cpp/src/io/csv/csv_reader.cu

```diff
@@ -79,7 +78,7 @@ typedef struct raw_csv_ {
 
     long				num_bytes;		// host: the number of bytes in the data
     long				num_bits;		// host: the number of 64-bit bitmaps (different than valid)
-	unsigned long long 	num_records;  	// host: number of records (per column)
+	unsigned long long 	num_records;  	// host: number of records loaded into device memory, and then number of records to read
```

Comment:
All variables that ultimately refer to the number of elements in a gdf_column should use gdf_size_type.

```suggestion
	gdf_size_type 	num_records;  	// host: number of records loaded into device memory, and then number of records to read
```